src/backend/templates/views/markdown/privacy.md

```diff
@@ -37,16 +27,51 @@ When you create a new account, you agree to our collection of this User Personal
  - We will use your email address to communicate with you for the purposes of marketing, if you’ve said that’s okay.
  - We will use your email address to communicate with you about important updates to our service.
  - We use your User Personal Information for internal purposes, such as moderation, maintaining logs for security purposes and for legal documentation.
+   - We retain certain technical identifiers (such as IP addresses and device/browser fingerprints) and 3rd party identifiers (such as SteamID or gog.com ID) where necessary to detect repeat abuse, prevent ban evasion, or investigate serious violations of our Terms of Service. This is based on our legitimate interest in maintaining the integrity and security of our community. These identifiers are not used for profiling, advertising, or user behavior analysis unrelated to security or moderation.
```

Comment:
How do deleted users get processed in analysis such as:

- https://forum.faforever.com/topic/9118/faf-statistics-megathread-2-statistics-boogaloo/2

They end up being anonymized, but still recognizable between data samples?

Reply:
We cannot delete them from the database technically. We could hide them from API querying though, but that needs some extra work.

Reply:
No, correction we can't even do that. It would break queries with other players involved. So this data will stay and we are allowed to keep it if other users are involved.